Makefile

```diff
@@ -116,9 +119,16 @@ run-slice:
 ifeq (,$(wildcard nodelogs))
 	mkdir nodelogs
 endif
-	$(PRIME_CMD) >> $(PRIME_LOG_FILE) 2>&1 &
-	$(REGION_CMD) >> $(REGION_LOG_FILE) 2>&1 &
-	$(ZONE_CMD) >> $(ZONE_LOG_FILE) 2>&1 &
+ifeq ($(filter $(REGION),0 1 2),)
```

Comment:
i saw this change in the other PR, please rebase properly

Reply:
wasn't this supposed to be removed from this PR?

Reply:
yeah needs to be removed